Makefile

```diff
@@ -1,5 +1,26 @@
 SHELL := /bin/bash
 
+SEC_GITHUB_PLUGIN_VERSION=2.3.0
+
+TMPDIR := $(shell mktemp -d)
+secrets_github_sig := "${TMPDIR}/plugin-secrets-github/SHA256SUMS.sig"
+secrets_github_sums := "${TMPDIR}/plugin-secrets-github/SHA256SUMS"
+
+# update-secrets-github-plugin will download SHA256SUMS and sig files for given version
+# verify its signature and then parse SUM file to extract plugin binary's SHA256
+# that SHA value will be used by script to register plugin with vault.
+# vault does verification of actual plugin binary and registered SHA of the plugin
+.PHONY: update-secrets-github-plugin
+update-secrets-github-plugin:
+	@wget -q -O $(secrets_github_sig) https://github.com/martinbaillie/vault-plugin-secrets-github/releases/download/v${SEC_GITHUB_PLUGIN_VERSION}/SHA256SUMS.sig
```

Comment:
I'm going to be selfish here, can we please use `curl`?

I don't have `wget` locally, my reasoning being - we need `curl` for lots of things, its actively developed and `wget` is a small subset of `curl`'s features. Its good to narrow tech creep.

And in fact we already use `curl` in the same target.